### ISSUE TITLE
"Scroll to View" functionality of Keyboard focussing causes page to be zoomed in (and cannot be zoomed out to original scale)

### ISSUE THREAD

**Issue by cvan** [MozillaReality/FirefoxReality#755]
## Hardware
Oculus Go
eaa4fc5

## Steps to Reproduce
1. Load https://mdn.github.io/learning-area/html/forms/date-picker-fallback/index.html (by loading https://cvan.io/d/, clicking [`HTML tests`](https://mixedreality.mozilla.org/fxr-html-tests/), and clicking [`#2 Date Picker](https://mdn.github.io/learning-area/html/forms/date-picker-fallback/index.html)).
2. Press a field to bring the user's focus to it. Notice the zoom.
3. Dismiss the keyboard or press on the page outside the field. Notice the page doesn't zoom back out.

## Current Behavior
When an input field is focussed, the page is zoomed and cannot be zoomed back out (without refreshing the page, which could be a long form).

## Expected Behavior
When an input field is focussed, the page is zoomed and then zoomed back out.

**Comment by keianhzo** [comment]
I can reproduce this with the editable form field text but not with the date picker. The date picker does nothing for me and it makes sense as it should request a date prompt not the inputconnection and we don't have that wired yet. Or maybe is something that has changed in the GV version you are testing with?
